### DISCUSSION TITLE
Delivery Drivers for WooCommerce

### DISCUSSION THREAD

**Discussion by shubhamrane48** [inspireui/support#5221] [Working on Features]
recently lunched Fluxstore Delivery Boy application is supported only WCFM but you can use below plugin to integrate on single vendor applications or websites.

Plugin Link: https://wordpress.org/plugins/delivery-drivers-for-woocommerce/

**Comment by sundarbabu2** [comment]
I vote for this plugin. Kindly integrate this plugin. But I doubt, the delivery boy plugin will work in the backend admin panel only. Hence it should be working even now.

**Reply by Haniel5** [reply]
Hi, the new version 1.5.1 of FluxStore Delivery Boy app https://1.envato.market/4edvXL now also supports WooCommerce (single vendor). Thank you!

**Comment by Haniel5** [comment]
Hi friends, just sharing that our team is working on this feature and put heavy development effort to release it in the next couple of weeks 👍

**Comment by Haniel5** [comment]
Hi, the new version 1.5.1 of FluxStore Delivery Boy app https://1.envato.market/4edvXL now supports both WooCommerce (single vendor) and WCFM plugin (WC Lovers). Thank you!

**Reply by Yan-Aleman** [reply]
Cool 🔥🔥🔥
